src/test/java/org/jitsi/meet/test/pageobjects/web/ToolbarV2.java

```diff
@@ -0,0 +1,416 @@
+/*
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test.pageobjects.web;
+
+import org.jitsi.meet.test.util.*;
+import org.jitsi.meet.test.web.*;
+import org.openqa.selenium.*;
+
+import java.util.*;
+
+/**
+ * Represents the toolbar in a particular {@link WebParticipant}.
+ *
+ * @author Leonard Kim
+ */
+public class ToolbarV2 extends Toolbar {
+    /**
+     * Accessibility labels to be used as selectors for finding WebElements
+     * within the {@link ToolbarV2}.
```

Comment:
would it make sense to add a reference in this comment to the js file in which the values these are mapped to are located?

Reply:
I'm not sure. I see that as potentially a bug because the file can change and someone wouldn't know to update this comment.

Reply:
Could the simliar not be said for all selectors that have been used?

Reply:
My last comments actually doesn't address your comment at all. It provides something loosely connected to the issue of adding comments in jitsi-meet saying "this is used for torture." Let's say I do add the comments to jitsi-meet. I have declared aria-label in 5 separate files. Would you then want me to add a comment to each? An alternative would be to group the labels into a constants and add one comment there. Or make the assumption that people know aria-labels are used for both torture and screen readers.

Reply:
> Could the simliar not be said for all selectors that have been used?

Yeah I guess so


> Let's say I do add the comments to jitsi-meet. I have declared aria-label in 5 separate files. Would you then want me to add a comment to each? An alternative would be to group the labels into a constants and add one comment there. Or make the assumption that people know aria-labels are used for both torture and screen readers.

I guess I was just worried about the potential for updating one and forgetting to update another, but I see what you mean about it being sticky and there being lots of places to update.  Considering the tests will fail in this scenario anyway (and I think in a reasonably obvious way?) then I guess it feels ok as is.

Reply:
Maybe what you're liking is the idea of something like a data-qa attribute that's explicit. data-qa isn't some standard attribute, it stands out a bit, and it would have one use: to be used as selectors for torture tests. How this could apply to mobile I wouldn't know, but ignoring mobile I think that would address your concerns. Would you prefer I added data-qa (or a similar name) in additional to aria-label?